modules/openxr/extensions/openxr_extension_wrapper.cpp

```diff
@@ -185,6 +188,24 @@ void *OpenXRExtensionWrapper::set_frame_end_info_and_get_next_pointer(void *p_ne
 	return nullptr;
 }
 
+void OpenXRExtensionWrapper::set_view_configuration_view_count(uint32_t p_view_count) {
+	GDVIRTUAL_CALL(_set_view_configuration_view_count, p_view_count);
+}
```

Comment:
This is a kind of awkward API, but I can't think of a better way to do it.

Maybe we could make it somewhat clearer what this doing with a different name?

Like, `prepare_view_configuration()` which makes it clearer that this is called in preparation for calling `set_view_configuration_and_get_next_pointer()` on each view?